centreon/www/class/config-generate/abstract/object.class.php

```diff
@@ -35,6 +35,8 @@
 
 abstract class AbstractObject
 {
+    const VAULT_PATH_REGEX = '/^secret::\d+::/';
```

Comment:
```suggestion
    const VAULT_PATH_REGEX = '^secret::[^:]*::';
```
we didnt use the vault id anymore